spec/dummy_app/config/application.rb

```diff
@@ -31,7 +31,12 @@ class Application < Rails::Application
       if v >= Gem::Version.new("4.2") && v < Gem::Version.new("5.0.0.beta1")
         config.active_record.raise_in_transactional_callbacks = true
       end
-      if v >= Gem::Version.new("5.0.0.beta1")
+      if v >= Gem::Version.new("5.0.0.beta1") && v < Gem::Version.new("5.1")
+        config.active_record.belongs_to_required_by_default = true
+        config.active_record.time_zone_aware_types = [:datetime]
+      end
+      if v >= Gem::Version.new("5.1")
+        config.load_defaults 5.1
```

Comment:
This is bordering on annoyingly pedantic, I'm sure, but we should not use a `Float` here. `Float#to_s` is not predictable. Please use string `"5.1"`. Thanks.

```
5.1.to_s 
# => "5.1"
5.99999999999999999999999999999999999999999999999999999.to_s
=> "6.0"
```

I know, I know, it's not relevant, how likely are we to hit a floating point issue in a rails version number? But, I can't help it. :)